obal/data/roles/build_package/defaults/main.yaml

```diff
@@ -9,3 +9,4 @@ build_package_wait: true
 build_package_download_logs: false
 build_package_download_rpms: false
 build_package_waitrepo: true
+run_koji_whitelist_check: true
```

Comment:
We've been prefixing role-specific vars with the role name, so this would become `build_package_run_koji_whitelist_check` or `build_pacakge_koji_whitelist_check`